src/utils/common.js

```diff
@@ -17,4 +17,28 @@ const classNames = (names) => {
 
 }
 
-export { classNames };
\ No newline at end of file
+const webShare = (data) => {
```

Comment:
let's put this into separate file/module, doesn't seem very related to `classNames` in use

Reply:
Yes it's not related to classNames but they're both in common.js? Not sure if any of the current common functions deserve their own file?

Reply:
it's not like files are scarce, usually changes are easier to merge when unrelated stuff is in different file. Navigation is often also easier when you have `classNames.js`, `webShare.js` and can quickly jump by file name.

Reply:
Makes sense I'll fix it